docs/SMCWG-charter.md

```diff
@@ -0,0 +1,328 @@
+   * Names and email addresses of employees who will participate in the Working
+     Group and Forum as Member representatives.
+   * Emergency contact information for security issues related to certificate
+     trust.
+
+   Applicants that qualify as Certificate Issuers or Root Certificate Issuers
+   must supply the following additional information:
+
+   * URL of the current qualifying audit report.
+   * Links or references to issued end-entity certificates that demonstrate
+     them being treated as valid by a Certificate Consumer Member.
+
+   Such Applicant SHALL become a Member once the SMCWG has determined by
+   consensus among the Members during a SMCWG Meeting or Teleconference that
+   the Applicant meets all of the requirements above or, upon the request of
+   any Member of the SMCWG, by a Ballot among Members of the SMCWG. Acceptance
+   by consensus shall be determined or a Ballot of the Members shall be held
+   as soon as the Applicant indicates that it has presented all information
+   required above and has responded to all follow-up questions from the SMCWG
+   and the Member has complied with the requirements of Bylaw 5.5.
+
+   Certificate Issuer applicants that are not actively issuing S/MIME
+   certificates but otherwise meet these membership criteria MAY request to the
+   SMCWG that they be granted an invitation for Associate Member status in
+   accordance with Bylaw 3.1, subject to conditions designated by the SMCWG.
+
+2. Existing CAB Forum Members seeking to participate in the SMCWG, in accordance
+   to Bylaw 5.3.1(c), MUST formally declare their intent to participate in
+   writing and provide the SMCWG Chair with this declaration and evidence that
+   they meet the criteria set forth above. Such Applicants SHALL become Members
+   of the SMCWG as determined by consensus during a SMCWG Meeting or
+   Teleconference, or upon the request of any Member of the SMCWG, by a Ballot
+   among Members of the SMCWG.
+
+   In order to determine the composition of the initial set of SMCWG Members,
+   at least twenty-four (24) hours prior to the initial meeting of the SMCWG,
+   the SMCWG Chair SHALL publish a list of Members seeking to participate who
+   they determine meet the criteria set forth above. As the first order of
+   business at the first meeting of the SMCWG, those organizations on the
+   Chair’s list of proposed, qualifying Members SHALL vote to determine the
+   initial set of SMCWG Members.
+
+##### 4.2.3. Suspension and Cessation of Working Group Membership
+
+Members may resign from the SMCWG at any time. Resignation, suspension, or
+other termination of membership in the SMCWG does not prevent a Member from
+potentially having continuing obligations, under the Forum’s IPR Policy or any
+other document.
+
+A Certificate Consumer Member’s membership may be suspended if any of the
+following become true:
+
+1. it stops providing support for its membership-qualifying software
+   product;
```

Comment:
I understand the change from "updates" to "support" to be potentially problematic. The primary concern with the "updates" is that it is entirely possible that valuable and important software in an ecosystem may not be under active and ongoing development, in the form of new feature releases and updates being made. Especially prior to the production of any draft guidelines by the CWG, I feel a requirement for semi-frequent updates by Certificate Consumers isn't appropriate.
I do agree that this change loses meeting some of the goals of Ballot 205. Perhaps it would be acceptable to directly incorporate a requirement for both Certificate Consumers and Certificate Issuers that they participate in CWG and/or Forum activities in some way, representing their continued investment and interest in the ecosystem? Depending on the wording, I could see that also being problematic, but it would allow drafting language that directly addresses the goal, rather than indirectly.